perllib/FixMyStreet/Cobrand/Merton.pm

```diff
@@ -102,4 +102,8 @@ sub lookup_site_code_config { {
 
 sub cut_off_date { '2020-12-06' } # 1 yr prior to FMS Pro go-live
 
+sub abuse_reports_only { 1 }
+
+sub privacy_policy_url { '/about/privacy' }
```

Comment:
Might as well move this change into `Default.pm` so that the fix applies to all cobrands.